src/app/courses/components/courses-page/courses-page.component.ts

```diff
@@ -51,13 +54,10 @@ export class CoursesPageComponent implements OnInit {
   }
 
   loadMore() {
+    this.coursesCount += 10;
```

Comment:
why is this number hardcoded like that? Don't you have a variable for that somewhere?

Reply:
variable added